### PR TITLE
add Photo Gallery tile

### PR DESCRIPTION
**Pull request by claytonc**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -102,6 +102,15 @@ Mostra uma foto excepcional selecionada diariamente.
     :height: 577px
     :width: 867px
 
+Galeria de fotos
+^^^^^^^^^^^
```
Comment (hvelarde) [inline README.rst]: this markup is broken.

File: src/brasil/gov/tiles/tiles/photogallery.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+from collective.cover.widgets.textlinessortable import TextLinesSortableFieldWidget
+from plone.autoform import directives as form
+from plone.tiles.interfaces import ITileDataManager
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from zope import schema
+from zope.interface import implementer
+
+
+class IPhotoGalleryTile(IListTile):
+    """Displays a list photos gallery"""
```
Comment (hvelarde) [inline src/brasil/gov/tiles/tiles/photogallery.py]: the `docstring` is wrong.

File: src/brasil/gov/tiles/tiles/photogallery.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    form.omitted('tile_description')
+    form.no_omit(ITileEditForm, 'tile_description')
+
+    form.no_omit(ITileEditForm, 'uuids')
+    form.widget(uuids=TextLinesSortableFieldWidget)
+
+
+@implementer(IPhotoGalleryTile)
+class PhotoGalleryTile(ListTile):
+    """Displays a list of tags."""
```
Comment (hvelarde) [inline src/brasil/gov/tiles/tiles/photogallery.py]: the `docstring` is wrong.

File: src/brasil/gov/tiles/tiles/templates/photogallery.pt [hunk 4]
```diff
@@ -0,0 +1,50 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      i18n:domain="governo.planalto.portal">
```
Comment (hvelarde) [inline src/brasil/gov/tiles/tiles/templates/photogallery.pt]: the `i18n:domain` is wrong.

File: README.rst (hunk 1 above)
Comment (claytonc) [reply README.rst]: ok

Comment (claytonc) [reply README.rst]: ok

File: src/brasil/gov/tiles/tiles/photogallery.py (hunk 2 above)
Comment (claytonc) [reply src/brasil/gov/tiles/tiles/photogallery.py]: ok

File: src/brasil/gov/tiles/tiles/photogallery.py (hunk 3 above)
Comment (claytonc) [reply src/brasil/gov/tiles/tiles/photogallery.py]: ok

File: src/brasil/gov/tiles/tiles/templates/photogallery.pt (hunk 4 above)
Comment (claytonc) [reply src/brasil/gov/tiles/tiles/templates/photogallery.pt]: ok

**Pull request merged**